File: samples/typescript/services/SampleService.ts
import { 
	handleUnaryCall,
	handleClientStreamingCall,
	handleServerStreamingCall,
	handleBidiStreamingCall,
	sendUnaryData,
	ServerDuplexStream,
	ServerReadableStream,
	ServerUnaryCall,
	ServerWritableStream,
	status,
	UntypedHandleCall,
	Metadata
 } from '@grpc/grpc-js';
import { ServiceError } from './utils/error';
import { ApiType } from './utils/interfaces';
import { SampleServiceServer, SampleServiceService } from './protos/SampleService';
import * as SamplePackage from './protos/SamplePackage';

class SampleService implements SampleServiceServer, ApiType<UntypedHandleCall> {
	[method: string]: any;

	// @rpc @@webezyio - DO NOT REMOVE
	public sampleUnary: handleUnaryCall<SamplePackage.SampleMessage, SamplePackage.SampleMessage> = (
		call: ServerUnaryCall<SamplePackage.SampleMessage, SamplePackage.SampleMessage>,
		callback: sendUnaryData<SamplePackage.SampleMessage>
	) => {
		// let response:SamplePackage.SampleMessage = { SampleString: "SomeString" };
		// callback(null,response);
		callback(new ServiceError(status.UNIMPLEMENTED,"Method is not yet implemented"))
	}


}

export {
	SampleService,
	SampleServiceService
};

File: samples/python/services/SampleService.py
from google.protobuf.timestamp_pb2 import Timestamp
from typing import Iterator
import SampleService_pb2_grpc
import SamplePackage_pb2

class SampleService(SampleService_pb2_grpc.SampleServiceServicer):

	# @rpc @@webezyio - DO NOT REMOVE
	def SampleUnary(self, request: SamplePackage_pb2.SampleMessage, context) -> SamplePackage_pb2.SampleMessage:
		# response = SamplePackage_pb2.SampleMessage(SampleString=None)
		# return response

		super().SampleUnary(request, context)

